robotbase/src/main/java/frc/robot/commands/Posing/Pose_L2.java

```diff
@@ -0,0 +1,27 @@
+package frc.robot.commands.Posing;
+
+import edu.wpi.first.wpilibj2.command.ParallelCommandGroup;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import edu.wpi.first.wpilibj2.command.WaitUntilCommand;
+import frc.robot.Constants.ELEVATOR;
+import frc.robot.Constants.LATERATOR;
+import frc.robot.Robot;
+import frc.robot.commands.Elevator.ElevatorSetDistance;
+import frc.robot.commands.laterator.LateratorSetDistance;
+
+public class Pose_L2 extends SequentialCommandGroup {
```

Comment:
Use the term `prepose` instead of `pose`. The term `pose` is for robot positioning. We need to make sure we have clear, and separate terms for different robot functions. This will make the code easier to read and more maintainable.